src/main/java/org/dice/factcheckbenchmark/benchmark/rdf/RDFResource.java

```diff
@@ -0,0 +1,85 @@
+package org.dice.factcheckbenchmark.benchmark.rdf;
+
+import org.apache.jena.rdf.model.*;
+
+import java.util.*;
+
+/**
+ * @author DANISH AHMED on 3/22/2018
+ */
+public class RDFResource {
+
+    protected Resource resource;
+    protected Model model;
+    protected String label;
+    public String uri;
```

Comment:
True, uri and label were enough. But when it was implemented, I kept DefactoResource.java class from FactCheck in consideration for deciding attributes because, in the end, benchmark had to be running FactCheck system. And the other attributes we are using are utilized by FactCheck :)

Reply:
The benchmark should offer a certain API and FactCheck should implement the same API. So the benchmark should work with FactCheck, but you shouldn't assume that it will _only_ benchmark FactCheck :wink:

Reply:
Yup, I got the idea. Other systems can also be benchmarked on the current implementation. But I completely agree our implementation approach is not good, as it was considering FC while implementing. I will try to change the implementation. Hopefully, it will also resolve other comments.